core/domain/takeout_service_test.py

```diff
@@ -862,95 +862,74 @@ def test_export_data_nontrivial(self):
             self.GENERIC_MODEL_ID: {
                 'commit_type': self.COMMIT_TYPE,
                 'commit_message': self.COMMIT_MESSAGE,
-                'commit_cmds': self.COMMIT_CMDS
```

Comment:
Why are these deleted?

Reply:
Loading all the commit_cmds would cause the server to run out of memory, so @seanlip @BenHenning and I determined that we could safely remove these to resolve memory issues.